### PR TITLE
Add text for JWST section

### PR DESCRIPTION
**Pull request by larrybradley**

### CODE REVIEW

**Review by adrn — APPROVED** [review]
Thanks @larrybradley - this is great, and I think could be merged as is!

**Review by nmearl — APPROVED** [review]

File: src/ms.tex
Comment (pllim) [inline src/ms.tex:318]: @larrybradley , should we also list [synphot](https://github.com/spacetelescope/synphot_refactor) here, as it is also affiliated package like `gwcs`?

File: src/refs.bib
Comment (pllim) [inline src/refs.bib:14]: What's up with the indentation changes here? I am not familiar with the `bib` format.

Comment (larrybradley) [reply src/refs.bib:14]: I think my editor automatically changed tabs to spaces.

**Review by pllim — COMMENTED** [review]
Just a small comment and a nitpick, otherwise LGTM. I'll approve once comments are addressed.

p.s. Not sure how many telescopes will end up in the paper. If we get too popular, might need to revisit to shorten the text in the future; not sure.

File: src/ms.tex
Comment (larrybradley) [reply src/ms.tex]: I included `synphot` in the last commit.

Comment (pllim) [reply src/ms.tex]: Thanks! Any chance you can cite it consistently as the other packages? 😅

Comment (larrybradley) [reply src/ms.tex]: done!

File: src/refs.bib
Comment (pllim) [inline src/refs.bib:1803]: How was this author list generated? Vicki and some people are missing.

Comment (larrybradley) [reply src/refs.bib:1803]: From `zenodo`:  https://zenodo.org/record/5711522/export/hx

**Comment by pllim** [comment]
I cannot tell if the PDF build failure is related or not. @adrn , can you tell?

File: src/refs.bib
Comment (pllim) [reply src/refs.bib:1803]: Huh, it must have blindly copied over contributor list... Are we allowed to change author listing to deviate from Zenodo or has the ship sailed?

Comment (pllim) [reply src/refs.bib:1803]: Is this slightly better? I cannot tell. 🤷 

https://ui.adsabs.harvard.edu/abs/2018ascl.soft11001S/exportcitation

Comment (larrybradley) [reply src/refs.bib:1803]: It would be odd to include authors that aren't on the document that is referenced.  Keep in mind that the author list will be truncated in the paper.  Is there a better or another `synphot` reference?

Comment (pllim) [reply src/refs.bib:1803]: https://github.com/spacetelescope/synphot_refactor/blob/master/CITATION.md was the best I could come up with. I guess I'll just leave this to your good judgement...

Comment (larrybradley) [reply src/refs.bib:1803]: It's your package, so I'll leave it up to you.  Tell me what reference (or references) you want to use.

Comment (pllim) [reply src/refs.bib:1803]: Let's go with https://ui.adsabs.harvard.edu/abs/2018ascl.soft11001S/exportcitation then? Then it would be consistent with `CITATION.md` in the package and the author list is less controversial. Thanks!

Comment (pllim) [reply src/refs.bib:1803]: Wait a minute... how is this different from this below? Do we need two entries?

https://github.com/astropy/astropy-v5.0-paper/blob/2a62a2ee5326b969c397fb869ab56cc35cc56f0d/src/refs.bib#L1356-L1361

Comment (larrybradley) [reply src/refs.bib]: No.  I removed the duplicate entries and update the `synphot` reference to the one you suggested (ASCL).

**Review by pllim — APPROVED** [review]
Thanks!

**Pull request merged**
